README.md

```diff
@@ -1,9 +1,9 @@
 Rundeck WinRM Plugin
 --------------------
 
-This is a [Rundeck Node Execution plugin][1] that uses WinRM to connect to Windows and execute commands.  It uses the [OverThere Library][2] to provide the WinRM implementation, and uses Basic authentication over HTTPS.
+These are a [Rundeck Node Execution plugin and a Rundeck File Copier plugin][1] that uses WinRM to connect to Windows and execute commands.  It uses the [OverThere Library][2] to provide the WinRM implementation, and uses Basic authentication over HTTPS.
 
-[1]: http://rundeck.org/docs/manual/plugins.html#node-execution-plugins
+[1]: http://rundeck.org/2.3.2/plugins-user-guide/node-execution-plugins.html
```

Comment:
Keep links unrelated to the version of Rundeck, as this was visibly the intent of the text before your patch (see comment above for the appropriate links)

Reply:
Agreed